components/Form.js

```diff
@@ -0,0 +1,84 @@
+import styled from 'styled-components';
+import { nanoid } from 'nanoid';
+import { useRouter } from 'next/router';
```

Comment:
```suggestion
import { useRouter } from 'next/router';
```
Is the useRouter necessary?